app/assets/javascripts/templates/visualizations/controls/map-tools.hbs

```diff
@@ -0,0 +1,31 @@
+<div class="vis-ctrl-sublock">
+  <h4 class="vis-ctrl-subtitle">Heat Maps</h4>
+  <div>
+    <label for="hmap-by">Map by:</label>
+    <select id="hmap-by" class="form-control">
+      {{#each heatmaps}}
+        <option value="{{value}}" data-log-id="{{logId}}">{{label}}</option>
+      {{/each}}
+    </select>
+  </div>
+  <div>
+    <label for="map-radius-ctn">Heatmap Radius:</label>
+    <div id="map-radius-ctn" class="inline-block">
+      <input id="map-radius" value="{{radius}}"/>
+      <span>m</span>
```

Comment:
Should we change this to actually say "meters" instead?  I've never liked this stray 'm'.